### PR TITLE
chore(root): target ES2019

### PR DESCRIPTION
**Pull request by OttoAllmendinger**
For our node deployments this should not be an issue, since require Node 12 anyway.

The remaining question is browser support, which looks good for the 
features we care about:

Support: https://caniuse.com/?search=es2019

Issue: BG-40313

### CODE REVIEW

**Review by brandonblack — APPROVED** [review]
LGTM, but I don't know much about what browsers we need to target.

File: modules/core/package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
   },
   "license": "Apache-2.0",
   "engines": {
-    "node": ">=10.22.0 <17",
+    "node": ">=12 <17",
```
Comment (tylerlevine) [inline modules/core/package.json:19]: we can go to 14 here

**Review by tylerlevine — CHANGES_REQUESTED** [review]
minor fix to engines

File: modules/core/package.json (hunk 1 above)
Comment (OttoAllmendinger) [reply modules/core/package.json:19]: coming up

**Comment by tylerlevine** [comment]
also could you please add `engines` here and set to 14? https://github.com/BitGo/BitGoJS/blob/master/modules/account-lib/package.json

**Review by tylerlevine — APPROVED** [review]

**Pull request merged**